images/virtualization-artifact/pkg/controller/vmip/internal/iplease_handler.go

```diff
@@ -129,7 +131,7 @@ func (h IPLeaseHandler) createNewLease(ctx context.Context, state state.VMIPStat
 	if err != nil {
 		vmipStatus.Address = ""
 		msg := fmt.Sprintf("the VirtualMachineIP cannot be created: %s", err.Error())
-		h.logger.Info(msg)
+		log.Info(msg)
```

Comment:
Let's translate this into 'Error' as part of your PR

Reply:
This is not internal error to put it to Error level